System.Containers/Image.cs

```diff
@@ -0,0 +1,59 @@
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+namespace System.Containers;
+
+public class Image
+{
+    internal JsonNode manifest;
+    internal JsonNode config;
+
+    internal List<Layer> newLayers = new();
+
+    public Image(JsonNode manifest, JsonNode config)
+    {
+        this.manifest = manifest;
+        this.config = config;
+    }
+
+    public void AddLayer(Layer l)
```

Comment:
> since adding a duplicate layer would be a bit wasteful but generally harmless.

that's essentially what I was concerned with, but no big deal this early on